pxemgr/key.go

```diff
@@ -1,43 +1,19 @@
 package pxemgr
 
 import (
-	"fmt"
 	"os"
 	"path"
 )
 
 const (
-	vmlinuzFile      = "coreos_production_pxe.vmlinuz"
-	initrdFile       = "coreos_production_pxe_image.cpio.gz"
-	installImageFile = "coreos_production_image.bin.bz2"
-	qemuImageFile    = "coreos_production_qemu_usr_image.squashfs"
-	qemuKernelFile   = "coreos_production_qemu.vmlinuz"
+	vmlinuzFile = "coreos_production_pxe.vmlinuz"
+	initrdFile  = "coreos_production_pxe_image.cpio.gz"
```

Comment:
removing, we no more pull qemu images from the mayu file server, nor install  coreos into machines

pxe images are enough